### PR TITLE
fix paragraph appearance

### PR DESCRIPTION
**Pull request by ami-GS**
each paragraph showed raw '###".
fixed by placing new line after first image

### CODE REVIEW

**Comment by candysmurf** [comment]
@ami-GS, Your PR didn't fix the real issue. If you like to contribute, please add space between `##` and the title, markdown will display it nicely.

File: docs/PLUGIN_SIGNING.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ By default, the Snap daemon (snapteld) has plugin signing verification enabled.
 
 ##How it works
```
Comment (candysmurf) [inline docs/PLUGIN_SIGNING.md]: ## How it works

**Comment by ami-GS** [comment]
@candysmurf 
oops, I used "grip" for markdown appearance test and it showed nicely.
I fixed all. thanks for kindly reviewing:)

**Comment by candysmurf** [comment]
LGTM

**Comment by candysmurf** [comment]
@ami-GS, thanks for your work!

**Pull request merged**